### PR TITLE
*: clean up log

### PR DESCRIPTION
**Pull request by shenli**
@coocood @tiancaiamao PTAL

### CODE REVIEW

File: session.go [hunk 1]
```diff
@@ -300,7 +302,11 @@ func (s *session) Retry() error {
 		variable.GetSessionVars(s).RetryInfo.ResetOffset()
 		for _, sr := range nh.history {
 			st := sr.st
-			log.Warnf("Retry %s", st.OriginText())
+			txt := st.OriginText()
+			if len(txt) > sqlLogMaxLen {
+				txt = txt[:sqlLogMaxLen]
+			}
+			log.Warnf("Retry %s", txt)
```
Comment (coocood) [inline session.go]: Add length information.

**Comment by tiancaiamao** [comment]
LGTM

**Comment by shenli** [comment]
@coocood PTAL

**Comment by zimulala** [comment]
LGTM

**Pull request merged**